inc/class-command.php

```diff
@@ -537,8 +537,10 @@ protected function write_config_file() : bool {
 		// Set hosts.
 		$config['hosts'] = $this->get_project_hosts();
 
-		// Set the machine name.
-		$config['machine_name'] = $config['hosts'][0];
+		// Maybe set the machine name.
+		if ( 'default' === $config['machine_name'] ) {
```

Comment:
The `machine_name` index might not be set so this would throw a warning, it should just check if the setting exists and use that if it does.

Reply:
It should always be set because it's in the first config that's loaded: https://github.com/Chassis/Chassis/blob/main/config.yaml#L123 unless of course your wrapper unsets it and I'm not aware of that?

Reply:
Sorry, that was a bit of a terse response - the config here is purely coming from the PHP array above your edited lines, and the additional that may or may not be set in the `composer.json`. Once we've merged those it's used to generate the `config.local.yaml` file in the chassis directory.

Reply:
Ahh right! I wasn't sure if you parsed the config files first then reset them when you get to here.

That being said, this change gives me the ability to add I wanted to set the `machine_name` in my `composer.json` and this is an enhancement that's backwards compatible.

Reply:
I'm cool with the change but the code needs updating as `machine_name` might not be set yet per my first comment in this thread 😛